Increase sleep times in tests, for slower machines.

Patch by Mark H Weaver <mhw@netris.org>.

--- dmd/tests/basic.sh	2013-11-30 17:22:00.000000000 -0500
+++ dmd/tests/basic.sh	2014-02-16 02:18:34.036376953 -0500
@@ -46,7 +46,7 @@
 dmd -I -s "$socket" -c "$conf" -l "$log" &
 dmd_pid=$!
 
-sleep 1				# XXX: wait till it's up
+sleep 3				# XXX: wait till it's up
 kill -0 $dmd_pid
 test -S "$socket"
 $deco status dmd | grep -E '(Start.*dmd|Stop.*test)'
--- dmd/tests/respawn.sh	2013-12-01 16:50:37.000000000 -0500
+++ dmd/tests/respawn.sh	2014-02-16 02:19:16.958251953 -0500
@@ -39,7 +39,7 @@
     i=0
     while ! test -f "$1" && test $i -lt 20
     do
-	sleep 0.3
+	sleep 1
 	i=`expr $i + 1`
     done
     test -f "$1"
@@ -65,14 +65,14 @@
    #:provides '(test1)
    #:start (make-forkexec-constructor
 	    "$SHELL" "-c"
-	    "echo \$\$ > $service1_pid ; while true ; do sleep 1 ; done")
+	    "echo \$\$ > $service1_pid ; while true ; do sleep 3 ; done")
    #:stop  (make-kill-destructor)
    #:respawn? #t)
  (make <service>
    #:provides '(test2)
    #:start (make-forkexec-constructor
 	    "$SHELL" "-c"
-	    "echo \$\$ > $service2_pid ; while true ; do sleep 1 ; done")
+	    "echo \$\$ > $service2_pid ; while true ; do sleep 3 ; done")
    #:stop  (make-kill-destructor)
    #:respawn? #t))
 (start 'test1)
@@ -82,7 +82,7 @@
 dmd -I -s "$socket" -c "$conf" -l "$log" &
 dmd_pid=$!
 
-sleep 1				# XXX: wait till it's up
+sleep 3				# XXX: wait till it's up
 kill -0 $dmd_pid
 test -S "$socket"
 $deco status test1 | grep started
